src/Rules/Doctrine/ORM/EntityColumnRule.php

```diff
@@ -159,8 +159,8 @@ public function processNode(Node $node, Scope $scope): array
 				'Property %s::$%s type mapping mismatch: property can contain %s but database expects %s.',
 				$className,
 				$propertyName,
-				$propertyReadableType->describe(VerbosityLevel::typeOnly()),
-				$writableToDatabaseType->describe(VerbosityLevel::typeOnly())
+				$propertyReadableType->describe(VerbosityLevel::getRecommendedLevelByType($propertyReadableType)),
+				$writableToDatabaseType->describe(VerbosityLevel::getRecommendedLevelByType($writableToDatabaseType))
```

Comment:
Both levels should be determined like this: `VerbosityLevel::getRecommendedLevelByType($writableToDatabaseType, $propertyReadableType);`